hack/env.sh

```diff
@@ -49,15 +49,15 @@ export ARGOCD_OPERATOR_REGISTRY_BUILD_DIR=${ARGOCD_OPERATOR_REGISTRY_BUILD_DIR:-
 export ARGOCD_OPERATOR_REGISTRY_MANIFEST_DIR=${ARGOCD_OPERATOR_REGISTRY_MANIFEST_DIR:-"deploy/olm-catalog/${ARGOCD_OPERATOR_NAME}"}
 export ARGOCD_OPERATOR_REGISTRY_IMAGE_HOST_ORG=${ARGOCD_OPERATOR_REGISTRY_IMAGE_HOST_ORG:-${ARGOCD_OPERATOR_IMAGE_HOST_ORG}}
 export ARGOCD_OPERATOR_REGISTRY_IMAGE_NAME=${ARGOCD_OPERATOR_REGISTRY_IMAGE_NAME:-"${ARGOCD_OPERATOR_NAME}-registry"}
-export ARGOCD_OPERATOR_REGISTRY_IMAGE_REPO=${ARGOCD_OPERATOR_REGISTRY_IMAGE_REPO:-"quay.io/jmckind/${ARGOCD_OPERATOR_REGISTRY_IMAGE_NAME}"}
+export ARGOCD_OPERATOR_REGISTRY_IMAGE_REPO=${ARGOCD_OPERATOR_REGISTRY_IMAGE_REPO:-"${ARGOCD_OPERATOR_IMAGE_HOST_ORG}/${ARGOCD_OPERATOR_REGISTRY_IMAGE_NAME}"}
 export ARGOCD_OPERATOR_REGISTRY_IMAGE_TAG=${ARGOCD_OPERATOR_REGISTRY_IMAGE_TAG:-${ARGOCD_OPERATOR_BRANCH_NAME}}
 export ARGOCD_OPERATOR_REGISTRY_IMAGE=${ARGOCD_OPERATOR_REGISTRY_IMAGE:-"${ARGOCD_OPERATOR_REGISTRY_IMAGE_REPO}:${ARGOCD_OPERATOR_REGISTRY_IMAGE_TAG}"}
 
 # Operator util vars
 export ARGOCD_OPERATOR_UTIL_BUILD_DIR=${ARGOCD_OPERATOR_UTIL_BUILD_DIR:-"build/util"}
 export ARGOCD_OPERATOR_UTIL_IMAGE_HOST_ORG=${ARGOCD_OPERATOR_UTIL_IMAGE_HOST_ORG:-${ARGOCD_OPERATOR_IMAGE_HOST_ORG}}
 export ARGOCD_OPERATOR_UTIL_IMAGE_NAME=${ARGOCD_OPERATOR_UTIL_IMAGE_NAME:-"${ARGOCD_OPERATOR_NAME}-util"}
-export ARGOCD_OPERATOR_UTIL_IMAGE_REPO=${ARGOCD_OPERATOR_UTIL_IMAGE_REPO:-"quay.io/jmckind/${ARGOCD_OPERATOR_UTIL_IMAGE_NAME}"}
+export ARGOCD_OPERATOR_UTIL_IMAGE_REPO=${ARGOCD_OPERATOR_UTIL_IMAGE_REPO:-"${ARGOCD_OPERATOR_IMAGE_HOST_ORG}/${ARGOCD_OPERATOR_UTIL_IMAGE_NAME}"}
 export ARGOCD_OPERATOR_UTIL_IMAGE_TAG=${ARGOCD_OPERATOR_UTIL_IMAGE_TAG:-${ARGOCD_OPERATOR_BRANCH_NAME}}
 export ARGOCD_OPERATOR_UTIL_IMAGE=${ARGOCD_OPERATOR_UTIL_IMAGE:-"${ARGOCD_OPERATOR_UTIL_IMAGE_REPO}:${ARGOCD_OPERATOR_UTIL_IMAGE_TAG}"}
 
```

Comment:
I thought we were no longer using this shell.  Please convert it to a target in Makefile.

Reply:
I've cleaned up the scripts folder.